pkg/googlecloud/subscriber.go

```diff
@@ -61,6 +61,10 @@ type SubscriberConfig struct {
 	// ProjectID is the Google Cloud Engine project ID.
 	ProjectID string
 
+	// TopicProjectID is the underlying topic Google Cloud Engine project ID.
+	// This can be helpful when subscription is linked to a topic for another project.
```

Comment:
can you put information that this is optional, and ProjectID is used by default? thanks!